### PR TITLE
added carousel to home page

### PR DESCRIPTION
**Pull request by OksanaWeigandSuminski**
Hey @jrmcgarvey! Here is a carousel of images on the home page. Let me know if it's ok.

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,76 @@
...
+    <li data-target="#carouselExampleIndicators" data-slide-to="6"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="7"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="8"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="9"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="10"></li>
+  </ol>
+<div id="carouselExampleSlidesOnly" class="carousel slide" data-ride="carousel">
+  <div class="carousel-inner">
+    <div class="carousel-item active">
+      <img class="d-block w-100" src="/assets/1.jpg" alt="First slide" style="align-content: center;">
```
Comment (jrmcgarvey) [inline app/views/home/index.html.erb:28]: We need to put something descriptive for the alt values.  It should be a few words that says what the slide shows.  The purpose of the alt is to convey meaning to those who can't see the image.  "first slide" doesn't convey any meaning.

**Review by jrmcgarvey — CHANGES_REQUESTED** [review]
Oksana, I would like to see a couple of changes.

See the inline comment about use of the alt tags -- we need phrases that describe the image.  Also, we have to avoid including .DS_Store files, perhaps by something in .gitignore.  Those files don't really belong in the git repository, although I think we have some of them already.

**Comment by OksanaWeigandSuminski** [comment]
I made changes. Please take a look @jrmcgarvey.

**Review by jrmcgarvey — APPROVED** [review]
Looks ok now

**Pull request merged**